lib/api/core/models/security/profile.js

```diff
@@ -10,6 +10,23 @@ function Profile () {
   this.roles = [];
 }
 
+/**
+ * @param {Kuzzle} kuzzle
+ *
+ * @return {Promise}
+ */
+
+Profile.prototype.getRoles = function (kuzzle) {
+  if (!this._roles) {
+    return kuzzle.repositories.role.loadRoles(this.roles)
```

Comment:
nope, just a naming problem.
getRoles use the this.roles var (which stores the roles list as an object like ```{_id:'roleId', restrictedTo:{}...}```) retrieves them via the loadRoles method, then keep them in their full business representation into this._roles.

I am currently renaming the this.roles into this.policies